packages/store-ui/src/molecules/PriceRange/PriceRange.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+
+import type { PriceProps } from '../../atoms/Price'
+import Price from '../../atoms/Price'
+import type { SliderProps } from '../../atoms/Slider'
+import Slider from '../../atoms/Slider'
+
+export type PriceRangeProps = SliderProps & {
+  /**
+   * Formatter function that transforms the raw price value and render the result.
+   */
+  formatter: PriceProps['formatter']
+  /**
+   * Returns the value of element's class content attribute.
+   */
+  className?: string
+}
+
+const PriceRange = ({
+  onChange,
+  testId = 'store-price-range',
+  className,
+  formatter,
+  ...props
```

Comment:
Just to avoid accessing the `props` object without deconstructing it
```suggestion
  min: initialMin,
  max: initialMax,
  ...props
```